doc/user/content/ingest-data/_index.md

```diff
@@ -0,0 +1,244 @@
+have the same ingestion timestamp.
+
+### Duration
+
+The snapshotting operation duration depends on the snapshot dataset size and the
+size of the Materialize cluster that is hosting the source. For very large
+sources that contain hundreds of GB of data, it can take up to an hour or even
+several hours to complete.
+
+{{% tip %}}
+
+- If possible, schedule creating new sources during off-peak hours to mitigate
+  the impact of snapshotting on both the upstream system and the Materialize
+  cluster.
+
+- Consider using a larger cluster size during snapshotting. Once the
+  snapshottingis complete, you can downsize the cluster to align with the volume
+  of changes being replicated from your upstream in steady-state.
+
+- See also [Best practices](#best-practices).
+
+{{% /tip %}}
+
+If you create your source from the Materialize Console, the overview page for
+the source displays the snapshotting progress. Alternatively, you can run a
+query to monitor its progress. See [Monitoring the snapshotting progress](/ingest-data/monitoring-data-ingestion/#monitoring-the-snapshotting-progress).
+
+### CPU and memory utilization
+
+Snapshotting may require more compute resources from the Materialize cluster
+than steady-state. If there are signs of resource exhaustion (that is, the
+cluster restarts because it ran out of memory), resize the cluster.
+
+{{% tip %}}
+Consider using a larger cluster size during snapshotting. Once the snapshotting
+operation is complete, you can downsize your source cluster to align with the
+volume of changes being replicated from your upstream in steady-state.
+{{% /tip %}}
+
+### Queries during snapshotting
+
+While a source is snapshotting, the source (and the associated subsources)
+cannot serve queries. That is, queries issued to the snapshotting source (and
+its subsources) will return after the snapshotting completes (unless the user
+breaks out of the query). If the user does not break out of the query, the
+returned query results will reflect the data from the snapshot.
+
+## Running/steady-state
+
+Once snapshotting completes, Materialize transitions to Running state. During
+this state, Materialize continually ingests changes from the upstream system.
+
+### Queries during steady-state
+
+Although Materialize is continually ingesting changes from the upstream system,
+depending on the volume of the upstream changes, Materialize may lag behind the
+upstream system. If the lag is significant, queries may block until Materialize
+has caught up sufficiently with the upstream system when using the default
+[isolation level](/get-started/isolation-level/) of [strict
+serializability](/get-started/isolation-level/#strict-serializable).
+
+In the Materialize Console, you can see a source's data freshness from the
+**Data Explorer** screen. Alternatively, you can run a query to monitor the lag.
+See [Monitoring the snapshotting progress](/ingest-data/monitoring-data-ingestion).
+
+## Rehydration
```

Comment:
We no longer use the term "rehydration", only "hydration".

Reply:
As for hydration vs rehydration ... I did ask about that since you had mentioned it to me in an earlier pr.  There seemed to be some ambiguity as to how we use the terms.

Reply:
I could go with Hydration and in that first sentence, add "(sometimes referred to as rehydration)."

Reply:
> (sometimes referred to as rehydration).

We should have a single term for user-facing documentation (or anything), and we've previously agreed that it should be simply _hydration_. Sources are either snapshotting (from the external system) or hydrating (from storage), compute objects just hydrate. No need to differentiate and potentially confuse users, since this is just our internal lingo coming through.